File: Arreglos/C/Columnas_DI_I.c
#define DIMENSION 5

#include <stdio.h>

int main() {
    int arreglo[DIMENSION][DIMENSION];
        int filas = 0, columnas = 0;

        for (columnas = 0; columnas < DIMENSION; columnas++) {

            for (filas = 0; filas < DIMENSION; filas++) {

                if (columnas + filas >= DIMENSION - 1) {
                    printf("(%d,%d) ", columnas, filas);
                }
                else{
                    printf("      ");
                }
            }

            printf("\n");
        }
    return 0;
}

File: Arreglos/Python/Columnas_DI_I.py
'''
Author: "Space Team"
Date: 12/02/2022
Description: Printing the upper diagonal of a matrix traversed by rows
'''

#Input
range = [0,1,2,3,4]
Dimension = 5
filas = 0
columnas = 0

#Algorithm
for columnas in range:
    Dimension = Dimension-1  
    for filas in range:
        if filas < Dimension:
            print(f"      ", end= "")       

    for filas in range:
        if filas >= Dimension:
            print(f"({filas},{columnas}) ", end= "") 
       
    print(" ")   

#Output       